src/physics.rs

```diff
@@ -105,7 +100,7 @@ impl Physics {
         terrain: &TerrainBody,
     ) {
         //Note: real world power is -11, but our scales are different
-        const GRAVITY: f32 = 6.6743e-8;
+        const GRAVITY: f32 = 1e-3;
```

Comment:
For some reason the working magic number is 1e-3, at least my demo cubes fall and collide more or less realistically